code/game/objects/items/tools/weldingtool.dm

```diff
@@ -33,14 +33,21 @@
 	wound_bonus = 10
 	bare_wound_bonus = 15
 	custom_materials = list(/datum/material/iron=70, /datum/material/glass=30)
-	///Whether the welding tool is on or off.
+	/// Whether the welding tool is on or off.
 	var/welding = FALSE
-	var/status = TRUE //Whether the welder is secured or unsecured (able to attach rods to it to make a flamethrower)
-	var/max_fuel = 20 //The max amount of fuel the welder can hold
-	var/change_icons = 1
-	var/can_off_process = 0
-	var/burned_fuel_for = 0 //when fuel was last removed
+	/// Whether the welder is secured or unsecured (able to attach rods to it to make a flamethrower)
+	var/status = TRUE
+	/// The max amount of fuel the welder can hold
+	var/max_fuel = 20
+	/// Whether or not we're changing the icon based on fuel left.
+	var/change_icons = TRUE
+	/// Used in process(), dictates whether or not we're calling STOP_PROCESSING whilst we're not welding.
+	var/can_off_process = FALSE
+	/// When fuel was last removed.
+	var/burned_fuel_for = 0
+	/// Activation sound path.
 	var/acti_sound = 'sound/items/welderactivate.ogg'
```

Comment:
theres no point in making a comment to document the name of something, these variables should be renamed to activation_sound and deactivation_sound if youre going to do it